lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -2992,6 +3007,31 @@ Optional<ActorIsolation> ConformanceChecker::checkActorIsolation(
       break;
     }
 
+    if (refResult.isolation) {
+      /// A property inside a class/actor may be unable to be witness to the
+      /// requirement, if it has a setter, since those cannot be async. Structs
+      /// are except from this since they don't need isolation.
```

Comment:
```suggestion
      /// A property inside a class/actor may be unable to witness the
      /// requirement, such as if it has a setter since those cannot be async.
      /// Structs are exempt from that since they don't need isolation.
```